### DISCUSSION TITLE
New Ideas Cafe is live..

### DISCUSSION THREAD

**Discussion by dtmf7133** [TiddlyWiki/TiddlyWiki5#6102] [General]
Hello everyone,

This time I will not ask Jeremy to include our TiddlyWiki site in the examples cause last time I did it I erased the project few days later.. oh my, searching for more luck this time.

However, you are everyone welcome to come in our Café and share your ideas or commenting on the existing ones:

http://newideas.cafe

Pls Enjoy!

Daniele Bonini
[par7133]

**Comment by Jermolene** [comment]
Hi @par7133 thanks for showing your site, it's always good to see people using TiddlyWiki. I generally don't include sites in the examples section unless there's something distinctive and notable about them that might be useful to the broader community.

**Comment by dtmf7133** [comment]
OK, thanks a lot to underline the qualities of our Café, I didn't expect...

Spare time, you are welcome to take a sit and join the conversation.

Daniele Bonini
[par7133]
